src/pages/MenuPage.js

```diff
@@ -106,14 +103,22 @@ export class MenuPage extends React.Component {
               text={navStrings.stocktakes}
               onPress={() => navigateTo('stocktakes', navStrings.stocktakes)}
             />
-            {isInAdminMode &&
-              <Button
-                style={globalStyles.menuButton}
-                textStyle={globalStyles.menuButtonText}
-                text="Realm Explorer"
-                onPress={() => navigateTo('realmExplorer', 'Database Contents')}
-              />
-            }
+            {isInAdminMode && (
+              <View>
+                <Button
+                  style={globalStyles.menuButton}
+                  textStyle={globalStyles.menuButtonText}
+                  text="Realm Explorer"
+                  onPress={() => navigateTo('realmExplorer', 'Database Contents')}
+                />
+                <Button
```

Comment:
I am not sure this will fit on some devices, I remember seeing fourth button in one column disappearing of screen